src/goSuggest.ts

```diff
@@ -183,6 +185,18 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 								item.insertText = new vscode.SnippetString(suggest.name + '(' + paramSnippets.join(', ') + ')');
 							}
 
+							if (wordAtPosition && wordAtPosition.start.character === 0 &&
+								suggest.class === 'type' && !goBuiltinTypes.has(suggest.name)) {
+								let prefix = 'func (' + suggest.name[0].toLowerCase() + ' *' + suggest.name + ')';
+								let auxItem = new vscode.CompletionItem(suggest.name + ' method', vscode.CompletionItemKind.Snippet);
+								auxItem.label = suggest.name + ' (new method)';
+								auxItem.detail = prefix + '...';
+								auxItem.sortText = 'a';
+								let snippet = prefix + ' ${1:name}(${2:params}) ${3:retval} \{\n\t$0\n\}';
```

Comment:
Since `params` and `retval` can be empty, why not have them as simple tabstops instead of placeholders?

i.e `${1:methodName}(${2}) ${3}` instead of `${1:name}(${2:params}) ${3:retvall}`

Reply:
Done.